app/components/global_threads/thread_list/index.tsx

```diff
@@ -105,6 +107,14 @@ function ThreadList({haveUnreads, intl, isLoading, loadMoreThreads, listRef, mar
                 onEndReached={handleEndReached}
                 onEndReachedThreshold={2}
                 ref={listRef}
+                refreshControl={(
+                    <RefreshControl
+                        refreshing={isRefreshing}
+                        onRefresh={onRefresh}
+                        colors={[theme.centerChannelColor]}
+                        tintColor={theme.centerChannelColor}
+                    />
+                )}
```

Comment:
See how this is done in the postlist component, doing it like this will cause issues on Android